pyocd/core/coresight_target.py

```diff
@@ -86,8 +86,9 @@ def elf(self, filename):
             self._elf = None
         else:
             self._elf = ELFBinaryFile(filename, self.memory_map)
-            self.cores[0].elf = self._elf
-            self.cores[0].set_target_context(ElfReaderContext(self.cores[0].get_target_context(), self._elf))
+        for core_number in range(len(self.cores)):
+            self.cores[core_number].elf = self._elf
+            self.cores[core_number].set_target_context(ElfReaderContext(self.cores[core_number].get_target_context(), self._elf))
```

Comment:
This line should probably be changed to handle `self._elf` being None. Otherwise a None can be passed into `ElfReaderContext`.

Reply:
I indented it back to the way it was except for the loop